<?xml version="1.0" encoding="UTF-8"?>
<launch>
  
  <arg name="mode" default="$(env MOVEL_MODE)" doc="mode: [deployment, development]"/>

  <group if="$(eval mode == 'deployment')"> 
    <node pkg="velocity_setter" type="velocity_setter_node" name="velocity_setter_node">
      <rosparam file="$(find velocity_setter)/config/velocity_setter.yaml"/>
      <rosparam file="$(env HOME)/.config/movel/config/velocity_setter/config/velocity_setter.yaml"/>
    </node>
  </group>

  <group unless="$(eval mode == 'deployment')"> 
    <node pkg="velocity_setter" type="velocity_setter_node" name="velocity_setter_node">
      <rosparam file="$(find velocity_setter)/config/velocity_setter.yaml"/>
    </node>
  </group>

</launch>
